avivator/src/components/Controller/components/ChannelController.jsx

```diff
@@ -79,13 +104,24 @@ function ChannelController({
             ))}
           </Select>
         </Grid>
-        <Grid item>
+        <Grid item xs={1}>
           <ChannelOptions
-            handleRemoveChannel={handleRemoveChannel}
             handleColorSelect={handleColorSelect}
             disabled={isLoading}
+            setFull={() => setDomain('full')}
+            setMaxMin={() => setDomain('max/min')}
```

Comment:
Apparently MUI doesn't allow getting values on `MenuItem`? This is what I saw on stack exchange...maybe wrong.

Reply:
I think you resolved this issue with the data-attributes business.  I was having trouble access the text or something and it appeared to be a limitation with MUI, so if the data-attributes business works, shouldn't be an issue.